src/app/core/services/resolver.provider.js

```diff
@@ -8,6 +8,8 @@ export default function (app) {
         this.editProfilePagePrealoading = editProfilePagePrealoading;
         this.transactionsListPagePreloading = transactionsListPagePreloading;
         this.signInPagePreloading = signInPagePreloading;
+        this.adminPagePreloading = adminPagePreloading;
+        this.getUsersFromFirebase = getUsersFromFirebase;
         this.$get = function() { return this; };
```

Comment:
why return this? why do you need provider?

Reply:
We've got this file **resolver.provider**  from initial installation and started use it for asynchronous pages without thinking about type of service. But after your question we've had small talk and decided that it would better to change it to service **service** because we don't need provider for our goals. 
I can do it on a new branch later.

Reply:
do it right now!